rero_mef/utils.py

```
@@ -187,14 +187,16 @@ def _next_response(self):
             params["accessToken"] = access_token
 
         count = 0
+        # Retry up to 5 times if harvesting fails (network issues, server errors)
         while count < 5:
             try:
                 self.oai_response = self.sickle.harvest(**params)
                 self.oai_response.xml  # try to get xml
```

Ruff (0.14.14)

Found useless expression. Either assign it to a variable or remove it.

(B018)